Assets/Scripts/Game/DaggerfallMissile.cs

```diff
@@ -247,9 +230,12 @@ private void Start()
                 goModel.layer = gameObject.layer;
             }
 
-            // Ignore missile collision with caster (this is a different check to AOE targets)
-            if (caster)
-                Physics.IgnoreCollision(caster.GetComponent<Collider>(), this.GetComponent<Collider>());
+            string layerName;
+            if (caster && caster != GameManager.Instance.PlayerEntityBehaviour)
+                layerName = "SpellMissiles";
+            else
+                layerName = "Player";
```

Comment:
I feel a bit uneasy with possible consequences of changing player's missiles layer, can that be reverted or does it break something else?

Reply:
If the missile exists on the SpellMissiles layer then it can collide with the player. The current code base handles this by spawning missiles a few units in front of the player. However, this can cause spell missiles to go through a wall if you cast right next to one. If you're fast enough you can also get hit by your own arrows.